composer.json

```diff
@@ -8,7 +8,7 @@
         "imagine/imagine": "^1.3",
         "jajo/jsondb": "^3.0",
         "laravel/framework": "^10.1",
-        "rector/rector": "dev-main",
+        "rector/rector": "^0.15.20",
```

Comment:
dev-main should be used for faster feedback

Reply:
I'm testing here the tagged version, as this post is about a freshly tagged feature :+1: